openfl-workspace/xgb_higgs/plan/defaults

```diff
@@ -0,0 +1 @@
+../../workspace/plan/defaults
```

Comment:
I was under the impression that all plans inherit from OpenFL defaults without needing to mention the path. Is this required?

Reply:
I just tested it, your impression seems accurate. I removed this. It was included in other workspaces so I pulled it in by default